server/src/format.ts

```diff
@@ -9,15 +9,28 @@ export default async function format(
   try {
     return await exec(execPath, ['format', '-s', identWidth.toString()], text)
   } catch (errors) {
-    const errorMessage =
-      "prisma-fmt error'd during formatting. This was likely due to a syntax error. Please see Prisma Language Server output. To see the output, go to View > Output from the toolbar, then select 'Prisma Language Server' in the Output panel."
-    if (onError) {
-      onError(errorMessage)
+    if (
+      (typeof errors === 'string' || errors instanceof String) &&
+      errors.includes('ValidationError')
```

Comment:
Could this not also swallow _real_ errors that happen while trying to create a `ValidationError`?

Reply:
It could, yes. Depends on what the notification should include really.  The notification could always suggest it was likely due to a syntax error, but still include instructions on how to get a more detailed output. This might be a better way here.

Reply:
I agree.